src/lookup/query.ts

```diff
@@ -119,17 +120,18 @@ export async function query(
 
   let data;
   try {
-    data = await response.json();
+    data = normalizeVp(await response.json());
   } catch (e) {
     throw new Error(
       `The holder [${queryEndpoint}] did not return a valid JSON response: parsing failed with error ${e}`
     );
   }
   if (!isVerifiablePresentation(data)) {
     throw new Error(
-      `The holder [${queryEndpoint}] did not return a Verifiable Presentation: ${JSON.stringify(
-        data
-      )}`
+      `The holder [${queryEndpoint}] did not return a Verifiable Presentation: `
+      // ${JSON.stringify(
```

Comment:
Are we leaving this commented out so we can use it for local testing?

Reply:
Nope, it shouldn't have been commented out, that's an artifact of local tests I forgot to remove. Fixed in d957b12 .